modules/infra/submodules/storage/main.tf

```diff
@@ -19,6 +19,12 @@ locals {
       policy_json = data.aws_iam_policy_document.blobs.json
       arn         = aws_s3_bucket.blobs.arn
     }
+    costs = {
+      bucket_name = aws_s3_bucket.costs[0].bucket
```

Comment:
This is not gonna work when the resource count is 0. this should be a conditional merge based on `var.domino_cost.storage_enabled`